src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -390,6 +390,9 @@
         var X = this.ctrl.__subContext__;
 
         if ( ! X[key] ) {
+          if ( key == 'bareUserDAO' ) {
+            key = 'userDAO';
```

Comment:
Why can't the rule that generates the approvalRequest do this?

Reply:
We use bareUserDAO on the server and have to keep it. We need to use userDAO on the client for proper auth and permissioning, hence the bug.

Reply:
so the client uses the daokey to find the object for the reference view?